content/strategy-goals/strategy/cloud/iam-admin/index.md

```diff
@@ -1,5 +1,47 @@
 
-This team is newly formed and is currently defining goals.
+## Q2 Objective and Key Results
+
+### Problem Area: Align on Team Vision
+
+Today, the IAM and Admin Experience team is less than two weeks old and does not yet have a long-term mission or vision that aligns with the company vision.
+
+- Key Result 1: An updated handbook page for the IAM and Admin Experience team including mission, vision, and team working agreements.
+- Key Result 2: A GitHub board outlining the key efforts on our 12-month roadmap validated by customer product gaps.
+
+**Product and Engineering OKR Alignment:** This does not align to product & engineering OKRs. It is necessary due to the formation of a new team.
+
+### Problem Area: Unifying Feature Permissions
+
+Today, nearly all Sourcegraph features (Code insights, batch changes, search notebooks, etc) have unique, custom permissions models, creating confusion for users and preventing cross-feature use cases from being developed.
+
+- Key Result 1: A detailed product & technical plan for implementing a unified permissions model
+- Key Result 2: Implementation of the MVP for a unified permissions model
+- Key Result 3: Robust documentation for feature teams to adopt this updated permissions mode
+
+Within this area, in Q2 we are not working on:
+
+- Implementing RBAC for administrators
+- Updating features to respect our permissions, this will be owned by each feature team
+
+**Product and Engineering OKR Alignment:** Objective #2: Focus on time-to-value for a single use-case (Fixing Vulnerabilities). With a single permissions model, Sourcegraph feature teams will be able to enable cross-feature experiences for our customers to fix vulnerabilities. For example, setting up a code monitor to ensure a vulnerability is not re-introduced after fixing with a batch change.
+
+### Problem Area: Improve Admin Analytics
+
+Today, Sourcegraph Admins do not have a clear understanding of how their team is using (or not using) Sourcegraph. This information is valuable to help administrators (1) convey the value of Sourcegraph to executives and (2) make key decisions required to do their job effectively, such as monitoring remaining licenses.
+
+- Key Result 1: Identify and deliver an improved usage analytics experience for admins.
+
+**Product and Engineering OKR Alignment:** Objective #1: Faster Time-to-Value by enterprises. Today, enterprise admins have very little visibility into usage within their organization. They are forced to rely on the basic stats provided by Sourcegraph or anecdotal reviews from their team. Introducing more fine grained analytics will enable admins to easily quantify value to their team and aid Sourcegraph AE/CE in driving upsell conversations based on usage within the org.
+
+### Problem Area: Allow admins to control sign-ups via externally defined groups
```

Comment:
nit: other "Problem Area" use Title case but this one is using sentence case (we should use the latter according to our handbook, but being consistent is just as good enough).

Reply:
Fixed in next commit!